bridgesync/backfill_tx_sender.go

```diff
@@ -0,0 +1,272 @@
+package bridgesync
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/agglayer/aggkit/db"
+	"github.com/agglayer/aggkit/log"
+	"github.com/agglayer/aggkit/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	// Batch size for processing records
+	batchSize = 100
+)
+
+// BackfillTxnSender handles the backfilling of txn_sender field for bridge records
+type BackfillTxnSender struct {
+	db         *sql.DB
+	log        *log.Logger
+	client     types.EthClienter
+	bridgeAddr common.Address
+}
+
+// NewBackfillTxnSender creates a new instance of BackfillTxnSender
+func NewBackfillTxnSender(
+	dbPath string,
+	client types.EthClienter,
+	bridgeAddr common.Address,
+	logger *log.Logger,
+) (*BackfillTxnSender, error) {
+	database, err := db.NewSQLiteDB(dbPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to initialize database: %w", err)
+	}
+
+	return &BackfillTxnSender{
+		db:         database,
+		log:        logger,
+		client:     client,
+		bridgeAddr: bridgeAddr,
+	}, nil
+}
+
+// BackfillAll processes bridge table to backfill txn_sender field
+func (b *BackfillTxnSender) BackfillAll(ctx context.Context) error {
+	b.log.Info("Starting txn_sender backfilling process")
+
+	// Process bridge table
+	if err := b.backfillTable(ctx, "bridge"); err != nil {
```

Comment:
```suggestion
	if err := b.backfillTable(ctx, bridgeTableName); err != nil {
```

Reply:
linter was showing error in this case, so did this way